### ISSUE TITLE
prepare training data

### ISSUE THREAD

**Issue by gaozhangyang** [HuangYG123/CurricularFace#9]
Could you tell me how to prepare the training data and the 'refined_ms1m.txt'?  Thanks.

**Comment by KavyaBPadmesh** [comment]
Do you think this approach will work when we train using dataset little label noise?

**Comment by Dnekhaev** [comment]
Could anyone share results of comparison of ArcFace and CurricularFace on some public version of MS1M or any other datasets? In my first experience when I train on my own data, MV-Arc-Softmax with t=1.2 works better than ArcFace, but CurricularFace works worse than ArcFace. Seems that training list refined_ms1m.txt is essential to reproduce the results of the paper and make fair comparison.

**Comment by kwanking** [comment]
> Could you tell me how to prepare the training data and the 'refined_ms1m.txt'? Thanks.

looking forward!

**Comment by garbo-g** [comment]
Could you tell me how to prepare the training data and the 'refined_ms1m.txt'? Thanks.
